src/neighbors.cc

```diff
@@ -765,32 +765,34 @@ neighborsDigestSelect(PeerSelector *ps)
 
     storeKeyPublicByRequest(request);
 
-    for (size_t i = 0; i < Config.peers->size(); ++i) {
-        const auto p = &Config.peers->nextPeerToPing(i);
+    if (Config.peers) {
```

Comment:
Lots of formatting changes due to the need to indent a block. This is the only line actually added

Reply:
Looks like a dupe of #1502 to me. I would go with #1502 instead of this noisier and slightly more dangerous change.